unity-renderer/Assets/Scripts/MainScripts/DCL/Bridges/RealmsInfoBridge/RealmsInfoHandler.cs

```diff
@@ -22,9 +25,20 @@ public void Set(string json)
         public void Set(RealmsInfoModel newModel)
         {
             model = newModel;
-            DataStore.i.realm.playerRealm.Set(model.current?.Clone());
+            if (model.current != null && !string.IsNullOrEmpty(model.current.serverName))
```

Comment:
could be that combining it in one will work the same `!string.IsNullOrEmpty(model.current?.serverName)`